mkt/settings.py

```diff
@@ -1213,3 +1213,19 @@ def JINJA_CONFIG():
 
 # The UUID for Yogafire (Tarako Marketplace).
 YOGAFIRE_GUID = '3a8bbc18-3a91-4170-a25a-3ba2a8ae2c81'
+
+# Tell Fireplace to allow dev payment providers for desktop only.
+# The mozPay providers are still controlled by device settings.
+ALLOW_DEV_PAY_PROVIDERS = False
+
+# A map of pay providers for desktop payments. This is used by Fireplace.
+DEV_PAY_PROVIDERS = {
+    'mozilla-dev/payments/pay/v1':
+        'https://marketplace-dev.allizom.org/mozpay/?req={jwt}',
+    'mozilla-stage/payments/pay/v1':
+        'https://marketplace.allizom.org/mozpay/?req={jwt}',
+    'mozilla-alt/payments/pay/v1':
+        'https://payments-alt.allizom.org/mozpay/?req={jwt}',
+    'mozilla-local/payments/pay/v1':
+        'http://mp.dev/mozpay/?req={jwt}',
+}
```

Comment:
Yup, being able to access payments-alt payments on -dev seems unneccessary.

Would probably allow you to remove the boolean too.